bus-mapping/src/rpc.rs

```diff
@@ -121,6 +122,21 @@ impl<P: JsonRpcClient> GethClient<P> {
         Ok(resp.0.into_iter().map(|step| step.result).collect())
     }
 
+    /// Calls `eth_getCode` via JSON-RPC returning a contract code
+    pub async fn get_code_by_address(
+        &self,
+        contract_address: Address,
+    ) -> Result<Vec<u8>, Error> {
```

Comment:
I think `Vec<u8>` is appropriate here because the code is just list of bytes.
In `ethers` there is `Bytes` which is a wrapper over `Vec<u8>` that implements the deserialization as a hex string, but I think that once the value is deserialized, having a `Vec<u8>` is the best option here.